code/index.html

```diff
@@ -2,21 +2,93 @@
 <html>
   <head>
     <meta charset="utf-8" />
-    <title>Project 2: The News Site</title>
-    <link rel="stylesheet" href="./style.css" />
-    <!-- dont forget the meta tag for responsiveness -->
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Wedding Weekly</title>
+    <link rel="stylesheet" media="all" href="./style.css" type="text/css"/>
+    <link rel="icon" type="image/png" href="./Images/confetti.png">
   </head>
   <body>
     <header>
-      <!-- Some kind of logo and navbar -->
+      <div class="hamburger">
+        <div></div>
+        <div></div>
+        <div></div>
+      </div>
+      <nav>
+      <div class="left">
+        <a href class="left-links">Home</a>
+        <a href class="left-links">News</a>
+      </div>
+
+      <h1>Wedding Weekly &#127882</h1>
+      
+      <div class="right">
+        <a href class="right-links">About Us</a>
+        <a href class="right-links">Contact</a>
+      </div>
+      </nav>
+
+        
     </header>
+  
+      <div class="main">
+        <img id="mainphoto" src="./Images/coronawedding.jpg" alt="corona wedding">
```

Comment:
Nice to see that you're using the alt for the image but try describe it even more, here's some info:
https://www.w3schools.com/tags/att_img_alt.asp